modules/exploits/linux/http/apache_couchdb_cmd_exec.rb

```diff
@@ -69,6 +69,7 @@ def post_auth?
 
   def check
     get_version
+    return CheckCode::Unknown if @version < 0
```

Comment:
```suggestion
    return CheckCode::Unknown if @version.nil?
```

Should this be checking for `nil` instead of `< 0`?

The context is, comparing to `nil` will raise an exception:

```
>> nil < 0
(irb):5:in `<main>': undefined method `<' for nil:NilClass (NoMethodError)
```

Reply:
I think that's right. i'll resolve this one